package.json

```diff
@@ -22,7 +22,7 @@
     "bfx-api-node-core": "^1.5.8",
     "bfx-api-node-plugin-wd": "^1.0.4",
     "bfx-api-node-rest": "^4.3.1",
-    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.13.1",
+    "bfx-hf-algo": "git+https://github.com/avsek477/bfx-hf-algo.git#add-meta-in-algo-orders",
```

Comment:
needs to be changed after merge: https://github.com/bitfinexcom/bfx-hf-algo/pull/210